ci/requirements/doc.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - numba
   - numpy
   - numpydoc
-  - pandas<0.25  # Hack around https://github.com/pydata/xarray/issues/3369
+  - pandas
```

Comment:
Is #3369 fixed? I see no change in versioneer since I opened the ticket. Did you retest the RTD CI?